en/micro-integrator/docs/references/connectors/csv-module/csv-module-config.md

```diff
@@ -0,0 +1,559 @@
+    <td>Specify whether the CSV input has a header row</td>
+  </tr>
+  <tr>
+    <td>Separator</td>
+    <td>valueSeparator</td>
+    <td>Default : "," (comma)</td>
+    <td>Specify the separator to use in the CSV input.<br>To use tab as the separator, use the value tab to this property. To use space, use the value space.</td>
+  </tr>
+  <tr>
+    <td>Skip Headers</td>
+    <td>skipHeader</td>
+    <td>true, false</td>
+    <td>This is available only if the value of the <code>headerPresent</code> property is set to <code>Present</code>. The default value is <code>false</code>.</td>
+  </tr>
+  <tr>
+    <td>Skip Data Rows</td>
+    <td>dataRowsToSkip</td>
+    <td></td>
+    <td>Specify the number of data rows to skip in the CSV. The default is 0.<br>- If headerPresent is Present, then data rows are the rows excluding the first row.<br>- If <code>headerPresent</code> is <code>Absent</code>, then data rows are the rows starting from the first row.<br></td>
+  </tr>
+  <tr>
+    <td>Order by Column</td>
+    <td>orderByColumn</td>
+    <td></td>
+    <td>Order the CSV content by values of the given column. If you want to specify the column by column index, provide the index of the column (Indexes are starting from 1). <br>To specify the column by column name, give the column name within double quotes (e.g., "name"). <br>Specifying the column by column name works only if the value of the <code>headerPresent</code> property is <code>Present</code>.</td>
+  </tr>
+  <tr>
+    <td>Sort Columns</td>
+    <td>columnOrdering</td>
+    <td>Ascending, Descending</td>
+    <td>This option is enabled if the <code>orderByColumn</code> has a value. This determines whether the CSV should be ordered ascendingly or descendingly according to the given column.<br>The default value is <code>Ascending</code>.</td>
+  </tr>
+  <tr>
+    <td>Skip Columns</td>
+    <td>columnsToSkip</td>
+    <td></td>
+    <td>Specify columns to skip from the CSV payload. You can specify the columns as comma-separated values. <br>This property supports more complex queries also, you can find full specifications <a href="https://github.com/wso2-extensions/mediation-csv-module/blob/master/docs/column_skipper_query.md" target="_blank" rel="noopener noreferrer">here</a>.</td>
+  </tr>
+  <tr>
+    <td>Custom Header</td>
+    <td>customHeader</td>
+    <td></td>
+    <td>Set a custom header to the output CSV payload. If this property not specified, the header for the output CSV is determined as follows,<br>- If the value of the <code>headerPresent</code> is <code>Absent</code> , the output CSV would not have a header.<br>- If the value of the <code>headerPresent</code> is <code>Present</code> and <code>skipHeader</code> is set as <code>true</code>, output CSV would not have a header.<br>- If <code>headerPresent</code> is <code>Present</code> and <code>skipHeader</code> is set as <code>false</code>, output CSV would have the header of the input CSV.<br></td>
+  </tr>
+  <tr>
```

Comment:
This section is duplicated in line 30. Shall we remove this section, the section on line 30 seems more descriptive?

Reply:
Yes. But if a user specifically looks at this section, they won't know what was mentioned previously.